src/adapter/src/catalog/builtin.rs

```diff
@@ -1831,6 +1831,15 @@ pub static MZ_SUBSCRIPTIONS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
     is_retained_metrics_relation: false,
 });
 
+pub static MZ_SESSIONS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
```

Comment:
Would it be possible/useful to add when the session was started? Jan put that as a suggestion on the ticket.

Reply:
Possible? yes. Useful? I'm not entirely sure, though I could imagine that it's something people would be curious about. I've added it to the PR with some caveats in it's correctness. Let me know what you think.